platforms-serverless/netlify-ci/functions/index.js

```diff
@@ -33,6 +33,12 @@ exports.handler = async function (event, context, callback) {
 
   const deleteManyUsers = await client.user.deleteMany()
 
+  /*
+  const dirTree = require("directory-tree");
+  const tree = dirTree(process.env.LAMBDA_TASK_ROOT);
+  console.dir(tree, { depth: null });
+  */
+ 
```

Comment:
Do we want to keep these commented lines around?

Reply:
Simplest way to get an overview if our actual tests start to break (when Vercel or AWS Lambda changes the structure in some way) - so I considered this helpful.